api-docs/api-reference/methods/roles/get-list-effective-roles-assigned-to-a-user-v2.0.rst

```diff
@@ -0,0 +1,400 @@
+                "onRoleName": "roleName",
+                 "forTenants": [
+                     "t1",
+                     "t2"
+                 ],
+                "sources": [
+                  {
+                       "sourceType": "USER",
+                       "sourceId": "userId",
+                       "assignmentType": "DOMAIN"
+                       "forTenants": [
+                          "t1",
+                          "t2"
+                       ]
+                  },
+                  {
+                       "sourceType": "USERGROUP",
+                       "sourceId": "UserGroupAId",
+                       "assignmentType": "DOMAIN",
+                       "forTenants": [
+                          "t1",
+                          "t2"
+                       ]
+                  },
+                  {
+                       "sourceType": "USERGROUP",
+                       "sourceId": "UserGroupBId",
+                       "assignmentType": "TENANT",
+                       "forTenants": [
+                          "t1",
+                          "t2"
+                       ]
+                  },
+                  {
+                       "sourceType": "USERGROUP",
+                       "sourceId": "UserGroupCId",
+                       "assignmentType": "TENANT",
+                       "forTenants": [
+                          "t1"
+                       ]
+                  },
+                  {
+                       "sourceType": "SYSTEM",
+                       "sourceId": "IDENTITY",
+                       "assignmentType": "TENANT",
+                       "forTenants": [
+                          "t2"
+                       ]
+                  }
+                ]
+            }
+        ]
+    }
+ }
+
+**Across Domains Assignment Example Response**
+
+Assume:
+
+- d1t1 and d1t2 are tenants within the same domain (Domain 1).
+- d2t1 is a tenant in a different domain (Domain 2).
+- The user has the 'observer' role assigned on tenant d1t1, d1t2 in Domain 1
+  and on tenant d2t1 on Domain 2.
+
+JSON::
+
+ {
+    "RAX-AUTH:roleAssignments": {
+        "tenantAssignments": [
+            {
+                "onRole": "8899",
+                "onRoleName": "observer",
+                "forTenants": [
+                     "d1t1",
+                     "d1t2",
+                     "d2t1"
+                 ],
+                "sources": [
+                  {
+                       "sourceType": "USER",
+                       "sourceId": "userId",
+                       "assignmentType": "DOMAIN",
+                       "forTenants": [
+                         "d1t1",
+                         "d1t2",
+                         "d2t1"
+                     ]
+                  }
+                ]
+            }
+        ]
+    }
+ }
+
+**RCN Role Example Response**
+
+Assume:
+
+- The user's RCN contains the domain Domain 1 with the d1t1 and d1t2 tenants.
```

Comment:
for all three assumptions, put the tenants ``d1t1``, ``d1t2``, and ``d2t1`` between double backticks